README.md

```diff
@@ -44,9 +49,11 @@ It is expected that all the contributors, especially the `first time contributor
 7. Always look for **[OPEN TO CONTRIBUTE](https://github.com/IAmTamal/Milan/issues?q=is%3Aissue+is%3Aopen+label%3A%22%F0%9F%A4%A9+status+%3A++Up+for+Grab%22)** issues, you can go & get assigned.
 
 ## License 👮
-
+<p align="left"><img src="https://img.shields.io/github/license/IAmTamal/Milan?style=for-the-badge" /></p>
```

Comment:
Move this badge to the top as well, and add an anchor tag to it  - so that whenever I click on that badge I land in the license page.